resources/common.py

```diff
@@ -22,25 +25,34 @@ def build_s3_get_command(access, secret, region, bucket):
     env = build_aws_env_var(access, secret, region)
     return '{} {} s3 cp s3://{}/db.sql -'.format(env, aws, bucket)
 
+# get the env vars filled with the appropriate AWS credentials
 def build_aws_env_var(access, secret, region):
     env = 'AWS_ACCESS_KEY_ID="{}" AWS_SECRET_ACCESS_KEY="{}"'.format(access, secret)
     if region != '':
         env = '{} AWS_DEFAULT_REGION="{}"'.format(env, region)
```

Comment:
If the region isn't present, should we automatically fall back to something, or does do the AWS CLI tools do that themselves already?